Vizzuality/landgriffon
# Chore: Add landgriffon excel template

### General description

Changes the downloadable excel base template including the correct LandGriffon one

### Designs

_Link to the related design prototypes (if applicable)_

### Testing instructions

In the Vercel deployed version (available in this PR) navigate to the Admin section and click in the Download button on the left up corner. It should download the LandGriffon template instead of the Mars template

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
